code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -547,15 +546,6 @@ Make sure their actual health updates immediately.*/
 	amount *= 2 / 3
 	return ..()
 
-/mob/living/carbon/xenomorph/proc/handle_interference()
-	if(interference)
-		interference = max(interference-2, 0)
-
-	if(observed_xeno && observed_xeno.interference)
-		overwatch(observed_xeno,TRUE)
```

Comment:
This behavior wasn't replicated in the interference component when its added to stop their overwatch when they have lost their hivemind link. I also think it never worked the other way around either.